pkgs/by-name/lu/lunarvim/package.nix

```
@@ -0,0 +1,128 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, fetchurl
+, makeWrapper
+, cargo
+, curl
+, fd
+, fzf
+, git
+, gnumake
+, gnused
+, gnutar
+, gzip
+, neovim
+, nodejs
+, nodePackages
+, ripgrep
+, tree-sitter
+, unzip
+, nvimAlias ? false
+, viAlias ? false
+, vimAlias ? false
+, globalConfig ? ""
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "lunarvim";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "LunarVim";
+    repo = "LunarVim";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-z1Cw3wGpFDmlrAIy7rrjlMtzcW7a6HWSjI+asEDcGPA=";
+  };
+
+  # Pull in the fix for Nerd Fonts until the next release
+  patches = [(
+    fetchurl {
```

```suggestion
    fetchpatch {
```
please use fetchpatch instead